.gitattributes

```diff
@@ -2,4 +2,5 @@
 /.github export-ignore
 /.gitattributes export-ignore
 /scripts export-ignore
-*.css linguist-detectable=false
+/src/Illuminate/Foundation/** linguist-vendored
+/tests/Assets/__fixtures__/** linguist-detectable=false
```

Comment:
The modern way is actually `-linguist-detectable`
https://git-scm.com/docs/gitattributes